subprojects/model-core/build.gradle.kts

```diff
@@ -36,7 +36,7 @@ dependencies {
     testRuntimeOnly(project(":distributions-core")) {
         because("Tests instantiate DefaultClassLoaderRegistry which requires a 'gradle-plugins.properties' through DefaultPluginModuleRegistry")
     }
-    integTestDistributionRuntimeOnly(project(":distributions-core"))
+    integTestDistributionRuntimeOnly(project(":distributions-full"))
```

Comment:
why did this change?

Reply:
To keep `ModelRuleCachingIntegrationTest.groovy` working, I had to use other rules (cpp in this case). Do we have a better distribution that only contains native?

Reply:
Yes there is `distributions-native`. I think we should use that and then the change is fine.

The reason why this is not `distributions-jvm` right now, is because most of the "jvm" part is in core (where it should not be).

Reply:
Would be great if you can add a `because("...")` to the dependency declaration though that states that we need native for the `ModelRuleCachingIntegrationTest`.

Reply:
Done